paddle/fluid/imperative/execution_context.h

```diff
@@ -102,7 +102,10 @@ class DygraphExecutionContext : public framework::ExecutionContext {
   }
 
   bool HasAttr(const std::string& name) const override {
-    return attrs_.count(name) != 0 || default_attrs_.count(name) != 0;
+    if (attrs_.find(name) == attrs_.end()) {
```

Comment:
Is the logic before and after the modification the same?

Reply:
Yeah, they are the same logic